lib/Sema/TypeCheckStmt.cpp

```diff
@@ -1702,6 +1702,11 @@ class StmtChecker : public StmtVisitor<StmtChecker, Stmt*> {
     // Type-check the subject expression.
     Expr *subjectExpr = switchStmt->getSubjectExpr();
     auto resultTy = TypeChecker::typeCheckExpression(subjectExpr, DC);
+
+    // FIXME: What's happening here?
+    if (!resultTy)
```

Comment:
@hamishknight What does it mean when this is null? It's the only place that it happens. (And it's from a code completion test: `IDE/complete_stmt_controlling_expr.swift`)

Reply:
Ah, got it.